src/styles/core.css

```diff
@@ -0,0 +1,128 @@
+  --cv-primary-green-30: oklch(from var(--cv-primary-green) 30% calc(var(--cv-primary-green-base) + (sin(0.8 * pi) * c)) h);
+  --cv-primary-green-40: oklch(from var(--cv-primary-green) 40% calc(var(--cv-primary-green-base) + (sin(0.7 * pi) * c)) h);
+  --cv-primary-green-50: oklch(from var(--cv-primary-green) 50% calc(var(--cv-primary-green-base) + (sin(0.6 * pi) * c)) h);
+  --cv-primary-green-60: oklch(from var(--cv-primary-green) 60% calc(var(--cv-primary-green-base) + (sin(0.5 * pi) * c)) h);
+  --cv-primary-green-70: oklch(from var(--cv-primary-green) 70% calc(var(--cv-primary-green-base) + (sin(0.4 * pi) * c)) h);
+  --cv-primary-green-80: oklch(from var(--cv-primary-green) 80% calc(var(--cv-primary-green-base) + (sin(0.3 * pi) * c)) h);
+  --cv-primary-green-90: oklch(from var(--cv-primary-green) 90% calc(var(--cv-primary-green-base) + (sin(0.2 * pi) * c)) h);
+  --cv-primary-green-100: oklch(from var(--cv-primary-green) 100% calc(var(--cv-primary-green-base) + (sin(0.1 * pi) * c)) h);
+  
+  /* neutrals */
+  --cv-black: hsl(0deg 0% 0%);
+  --cv-white: hsl(0deg 0% 100%);
+}
+
+.cv-panel {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 18.75rem;
+  height: 100vh;
+  z-index: 999999;
+  transform: translateX(-100%);
+  transition: transform 0.3s ease;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, sans-serif;
+  box-shadow: 0.125rem 0 0.625rem var(--cv-black);
+  overflow-y: auto;
+}
+
+.cv-panel--visible {
+  transform: translateX(0);
+}
+
+/* Panel header */
+.cv-panel__header {
+  padding: 1rem 1.5rem 0.5rem;
+  border-bottom: 1px solid var(--cv-white);
+}
+
+.cv-panel__title {
+  margin: 0;
+  font-size: 1.125rem;
+  font-weight: 600;
+  color: var(--cv-white);
+}
+
+/* Panel content */
+.cv-panel__content {
+  padding: 1.5rem;
+}
+
+/* Features list & content */
+.cv-feature {
+  display: flex;
+  align-items: center;
+  gap: 0.75rem;
+  padding: 0.75rem;
+}
+
+.cv-feature__icon {
+  font-size: 1.25rem;
+  flex-shrink: 0;
+}
+
+.cv-feature__text {
+  color: var(--cv-white);
+  font-weight: 500;
+}
+
+/* Button */
+.cv-btn {
+  width: 100%;
+  padding: 1rem;
+  border: none;
+  border-radius: 0.5rem;
+  font-size: 1rem;
+  font-weight: 600;
+  cursor: pointer;
+  transition: all 0.2s ease;
+}
+
+.cv-btn--primary {
+  background: transparent;
+  color: var(--cv-white);
+  border: 2px solid var(--cv-white);
+}
+
+.cv-btn--primary:hover {
+  transform: translateY(-1px);
+  box-shadow: 0 4px 12px var(--cv-black);
+}
+
+.cv-btn--primary:active {
+  transform: translateY(0);
+}
+
+.cv-btn:disabled {
+  opacity: 0.6;
+  cursor: not-allowed;
+  transform: none;
+}
+
+/* Responsive design */
+@media (max-width: 480px) {
+  .cv-panel {
+    width: 100%;
+  }
+}
+
+/* High contrast mode support */
+@media (prefers-contrast: more) {
+  .cv-panel {
+    border: 2px solid var(--cv-black);
+  }
+}
```

Comment:
https://github.com/sparkbox/carbon-visualizer-extension/pull/2#discussion_r2414420141 (my comment was "outdated" but it's not really.